### DISCUSSION TITLE
Update one package.json in each PR

### DISCUSSION THREAD

**Discussion by devotox** [renovatebot/renovate#21333] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

In a monorepo I want to be able to ensure that updates only affect one package.json at a time. So even if the package or related package is in multiple package.json it creates multple PRs for them

### What you tried so far.

Looked through[ Configuration Options](https://docs.renovatebot.com/configuration-options) but to no avail

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
`"additionalBranchPrefix": "{{packageFileDir}}-"` will split each PR by the directory it exists in. You can use some `{{if}}` templating language too. See https://docs.renovatebot.com/templates/

**Reply by devotox** [reply]
Thank you!
